src/app/[locale]/commerce/orders/page.tsx

```
@@ -0,0 +1,358 @@
+    });
+  };
+
+  const formatCurrency = (amount: number, currency: { symbol: string; code: string }) => {
+    return `${currency.symbol}${amount.toFixed(2)}`;
+  };
+
+  if (loading) {
+    return (
+      <div className="p-6">
+        <div className="animate-pulse">
+          <div className="h-8 bg-gray-200 rounded w-1/4 mb-6"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)